docs/source/azure/configuration/containerhosting.rst

```diff
@@ -3,24 +3,18 @@
 Azure Container Hosting
 =======================
 
-The Azure provider can be run in a containerized environment using the official cloud custodian 
-`docker image <https://hub.docker.com/r/cloudcustodian/c7n>`_. The Azure Container Host knows 
-how to handle both periodic and event based policies.
+The Azure Container Host is an alternative execution mode for the cloud custodian azure provider.
+Running the Azure Container Host is done with the `official custodian docker image <https://hub.docker.com/r/cloudcustodian/c7n>`_
+by providing the custom entry point, ``/usr/local/bin/python3 -m c7n_azure.container_host.host``, 
+and custom configuration through environment variables.
 
-The Azure Container Host will read policies from an Azure Blob Container and, depending on their mode,
-will listen for event triggers or run them periodically. Event triggers are pulled from an event 
-queue within a storage account.
+The Azure Container Host will periodically scan azure blob storage for a set of custodian policies 
+to execute in either a periodic or event based mode against a target subscription. Once the
+Azure Container Host is deployed, any policies uploaded to blob storage are automatically loaded and
+running against an Azure Subscription. This makes it very easy to manage and run a large number of
+policies.
 
-To run the container, pass the docker command as `/usr/local/bin/python3 -m c7n_azure.container_host.host`
-and authenticate through environment variables (see :ref:`azure_authentication`). It is also important
-to make sure that the container host is authenticated as a contributor on the policy storage and a 
-message processor on the event queue.
-
-There are 3 important environment variables that are specific to the container host. 
-
-* `AZURE_EVENT_QUEUE_RESOURCE_ID`: The resource ID for a storage account for the event queue.
-* `AZURE_EVENT_QUEUE_NAME`: The name of the event queue. If this queue does not exist, it will be created.
-* `AZURE_CONTAINER_STORAGE`: The URI to an azure blob container that will hold all of the policies for this container host.
+.. image:: resources/container-host.png
```

Comment:
Should the image show multiple Azure subscriptions? One running container targets 1 - N subscriptions with the same policies right?

Reply:
One container targets a single subscription. To target multiple subscriptions, there would need to be one container host per subscription. But, they can each pull from the same storage account and run the same policies.

Although it does indeed feel like the docs are missing content on how to run against multiple subscriptions.

Reply:
Ahh so when we were testing it we had two pods running in the same cluster? One for each of the subscriptions

Probably a line somewhere in the introduction saying that a single container instance targets one subscription, but you can deploy many different ones to point at different subscriptions.

Wasn't there also the ability to provide the service principle information that the container should be executed with through the command line somewhere? I don't remember seeing it when I read through

Reply:
Yeah, we had two pods. I just pushed an update (with another diagram) that shows what it would look like to run against multiple subscriptions.

It is possible to pass the service principal, but it would also be possible to use a different authentication mechanism. I wanted to avoid mentioning service principals explicitly on this page since there are other (potentially better) options and didn't want to force people into thinking that was the only way to authenticate. So instead, I just link to the doc describing the different auth options.

The ACI tutorial uses a Managed Service Identity and should show how to set that up. The k8s tutorial will show how to authenticate too.